### DISCUSSION TITLE
Funding LNBits from the same Raspiblitz node (lnd)

### DISCUSSION THREAD

**Discussion by adevoss** [raspiblitz/raspiblitz#4191] [Q&A]
Trying out LNBits.

There are sats on my node.
Created a new LNBits user and a new LNBits wallet.

Now I want to fund the new LNBits wallet with the sats on my node.
How can this be done?
Don't know if this is a Raspiblitz question or a LNBits one. Starting here.

In the FAQ of LNBits (https://docs.lnbits.org/guide/faq.html) there is an answer to this question:
```
lnd.conf
allow-circular-route=1 
allow-self-payment=1
```
but after that change lnd won't restart. Does it has to be inserted in a specific place in lnd.conf? I just inserted these line at the beginning.

Setup:
Raspiblitz v1.10
LNBits version: 0.10.9
lnd: LND_GRPC 0.16.4-beta

**Comment by rootzoll** [comment]
Opened an issue #4192 to optimize RaspiBlitz/LNbits self-funding in the future.
